files/scripts/selinux/trivalent/trivalent.fc

```diff
@@ -0,0 +1,6 @@
+/usr/lib/trivalent/trivalent				--	gen_context(system_u:object_r:trivalent_exec_t,s0)
+/usr/lib/trivalent/chrome_crashpad_handler			--	gen_context(system_u:object_r:trivalent_exec_t,s0)
```

Comment:
Hm I don't think that should be launched in the `trivalent_t` domain considering it's just the crash handler, maybe add a TODO comment so in future we can confine it to its own domain